README.md

```diff
@@ -348,7 +340,6 @@ The following VS Code extensions use `wdio-vscode-service`:
 - [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) (27.8m downloads)
 - [DVC Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc) (11.2k downloads)
 - [Nx Console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console) (1.2m downloads)
-- [inlang – i18n supercharged](https://marketplace.visualstudio.com/items?itemName=inlang.vs-code-extension) (3k downloads)
```

Comment:
Why removing this? They still seem to use [WebdriverIO](https://github.com/opral/monorepo/blob/d5cb831d6b96aba3524589758830367d442923c3/inlang/source-code/ide-extension/package.json#L207)

Reply:
Ah, sorry, I checked out the README from your branch and didn't catch this or the section above in the diff. :facepalm: **Fixed**